sdk/go/pulumi/resource.go

```diff
@@ -192,6 +194,8 @@ type invokeOptions struct {
 	Parent Resource
 	// Provider is an optional provider resource to use for this invoke.
 	Provider ProviderResource
+	// URN is an optional URN of a previously-registered resource of this type to read from the engine for this invoke.
+	URN string
```

Comment:
Why is this needed on `invokeOptions`?

Reply:
Hmm, I guess this may be a copy-paste error. It doesn't seem like it's needed, so I'll rework this to remove it.